KarhooUISDK/Common/AddressBarMVP/KarhooAddressBarView.swift

```diff
@@ -39,6 +41,9 @@ public class KarhooAddressBarView: UIView, AddressBarView {
     private let verticalPadding: CGFloat
     private let horizontalPadding: CGFloat
     private let hidePickUpDestinationConnector: Bool
+    
+    private let destinationIconSize: CGFloat = 18.0
+
```

Comment:
I am not very a fan of two subsequent empty lines

Reply:
resolved